test/clj/quil/snippet.clj

```diff
@@ -14,9 +16,42 @@
 (def tests-in-set 50)
 (def current-test (atom -1))
 
+(defn skip-automated-compare? [snippet-name]
+  (contains? #{"current-frame-rate-target-frame-rate"
+               "noise"
+               "noise-detail"
+               "random-2d"
+               "random-3d"
+               "random"
+               "random-gaussian"
+               "resize-sketch"
+               "save"
+               "time-and-date"} snippet-name))
+
+(defn- compare-images [expected actual difference]
+  (let [{:keys [err]} (sh/sh "compare" "-metric" "mae" expected actual difference)
```

Comment:
Can you add comment here explaining what is "compare" and what is "mae"? Is that a command line tool that needs to be installed in order to run test?

Reply:
Added link to imagemagick docs. To be honest I'm not sure that `mae` is what we should be using. Saw it was used in `kamera` and went with that to start. The `threshold` values I chose are also quite arbitrary at this point. The idea is to get more experience with these parameters while upgrading p5 to 0.8.0 and processing to 3.5.3.

Don't like that we have to install an additional tool to run tests but didn't find any other alternatives in this space so far.

Reply:
done